cybereason/icon_cybereason/util/api.py

```diff
@@ -0,0 +1,72 @@
+import insightconnect_plugin_runtime
+import requests
+from insightconnect_plugin_runtime.exceptions import PluginException, ConnectionTestException
+import json
+from logging import Logger
+from typing import Optional
+
+
+class CybereasonAPI:
+    def __init__(self, hostname: str, port: int, username: str, password: str, logger: Logger):
+        self.base_url = f"https://{hostname}:{port}/"
+        self.username = username
+        self.password = password
+        self.headers = {"Content-Type": "application/json"}
+        self.session = requests.session()
+        self.logger = logger
+
+    def connect(self):
+        login_response = self.session.post(
+            f"{self.base_url}/login.html",
+            data={"username": self.username, "password": self.password},
+            verify=True,
+        )
+
+        if login_response.status_code not in range(200, 299):
+            raise ConnectionTestException(
```

Comment:
```suggestion
            raise PluginException(
```

Reply:
This method ("connect") is used only in connection. Should we change this to PluginException and then except in connection?